docs/source/api/pytorch/models.rst

```diff
@@ -42,13 +42,5 @@ Layers and model building blocks
    :members:
    :show-inheritance:
 
-Multi-objective models
```

Comment:
I would prefer not to rename this/remove it without doc version control

Reply:
I moved all multiobjective functionality into its own file, right now here: 
<img width="203" alt="image" src="https://github.com/user-attachments/assets/76318bbe-54a0-4b92-9824-d0123b2ec01b" />

I added a note at this point and a link.

Reply:
better? @MMathisLab

Reply:
great, thanks!